ckanext/publications_qld_theme/templates/organization/read_base.html

```diff
@@ -7,7 +7,7 @@
 <meta name="DCTERMS.publisher" scheme="AGLSTERMS.AglsAgent" content="corporateName=The State of Queensland; jurisdiction=Queensland" />
 <meta name="DCTERMS.creator" scheme="AGLSTERMS.GOLD" content="c=AU; o=The State of Queensland; ou={{ group_dict.display_name }}" />
 <meta name="DCTERMS.created" content="{{ h.reformat_date_string(group_dict['created']) }}" />
-<meta name="DCTERMS.description" content="{{ group_dict['description'] or '' }}" />
```

Comment:
should we not limit description to a number of char's as they can be Huge.

https://www.w3.org/Submission/prism/prism-dc.html#_Toc336961559 

I think we should limit it to 500 char, html escaped string

Reply:
HTML escaping is built in.

I've added 500-character truncation.